Restructure Service Brokers documentation (#2667)
kyma-project/kyma: docs/kyma/docs/03-01-components.md

## Service Catalog

> The Service Catalog lists all of the services available to Kyma users through the registered [Service Brokers](/components/service-catalog#service-brokers-service-brokers). Using the Service Catalog, you can provision new services in the

Reviewer: ```suggestion
The Service Catalog lists all of the services available to Kyma users through the registered [Service Brokers](/components/service-catalog#service-brokers-service-brokers). Use the Service Catalog to provision new services in the
```

Kyma security enforces RBAC (Role Based Access Control) in the cluster. [Dex](https://github.com/coreos/dex) handles the identity management and identity provider integration. It allows you to integrate any [OpenID Connect](https://openid.net/connect/) or SAML2-compliant identity provider with Kyma using [connectors](https://github.com/coreos/dex#connectors). Additionally, Dex provides a static user store which gives you more flexibility when managing access to your cluster.   

## Helm Broker

> The Helm Broker is a Service Broker which runs in the Kyma cluster and deploys Kubernetes native resources using [Helm](https://github.com/kubernetes/helm) and Kyma bundles. A bundle is an abstraction layer over a Helm chart which allows you to represent it as a ClusterServiceClass in the Service Catalog. Using bundles, you can install the [GCP Broker](/components/service-catalog#service-brokers-gcp-broker) and the [Azure Service Broker](/components/service-catalog#service-brokers-azure-service-broker) in Kyma.

Reviewer: ```suggestion
The Helm Broker is a Service Broker which runs in the Kyma cluster and deploys Kubernetes native resources using [Helm](https://github.com/kubernetes/helm) and Kyma bundles. A bundle is an abstraction layer over a Helm chart which allows you to represent it as a ClusterServiceClass in the Service Catalog. Use bundles to install the [GCP Broker](/components/service-catalog#service-brokers-gcp-broker) and the [Azure Service Broker](/components/service-catalog#service-brokers-azure-service-broker) in Kyma.
```
